build/docker-compose.build-template.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  default:
+    image: ${IMAGE_REGISTRY:+${IMAGE_REGISTRY}/}${IMAGE_NAMESPACE?}:/${IMAGE_NAME?}:${IMAGE_VERSION?}
+    build:
+      tags:
+        - ${IMAGE_REGISTRY:+${IMAGE_REGISTRY}/}${IMAGE_NAMESPACE?}/${IMAGE_NAME?}:latest
```

Comment:
We do not tag `:latest` on dockerhub at the moment. How can we tag it when building (in order to access the latest builds while testing) but not push `:latest` on publish?

Reply:
Indeed. The current Github Actions `_build_test.yml` step overwrites these tags and just pushes the constrcuted `dev<gh-run-id>` tag as before. The part about actually publishing these to Dockerhub is actually WIP. At this point we have several options I kinda want to evaluate which one we choose as/when we finish settling decisions about how we want to run our release process as of this work... This file also is extraneous and I will be removing it, so will resolve the convo, but for sure by the time we're pushing to Dockerhub things will likely look different and we certainly won't push the `latest` tag. Promise.